examples/build.gradle

```diff
@@ -25,6 +25,7 @@ repositories {
 def grpcVersion = '1.5.0-SNAPSHOT' // CURRENT_GRPC_VERSION
 
 dependencies {
+  compile "com.google.api.grpc:grpc-google-common-protos:0.1.12"
```

Comment:
I think this should use proto-google-common-protos, not grpc-google-common-protos. See https://github.com/grpc/grpc-java/issues/2961#issuecomment-298370880